docs/_partner_api/elements.md

```diff
@@ -0,0 +1,82 @@
+```
+
+Want to embed multiple Zap Templates? Include each of their IDs in a comma separated list, such as:
+
+```javascript
+<script type="text/javascript" src="https://zapier.com/apps/embed/widget.js?guided_zaps=1234,9876,3456"></script>
+```
+
+You can additionally use these options to customize your Zap Template embeds:
+- `limit=10` to set the number of Zaps to display; use any number you want
+- `theme=dark` for a dark-colored embed, instead of the default light background
+- `categories=CATEGORYNAME` to show only Zap Templates with apps from a specific category, with `CATEGORYNAME` replaced with the name of a category from [Zapier’s App Directory](https://zapier.com/apps/)
+- `categories=-CATEGORYNAME` to _not_ show Zap Templates with apps from a specific category
+- `services=APP,APP` to show only Zap Templates between two (or more) apps 
+- `services=-APP,-APP` to exclude specific apps
+- `borderColor=%23333` to set the color of the borders to any css value or “none”. Use "%23" in place of "#" for HEX values.
+- `backgroundColor=green` to set the background of each row to any css value including “transparent”
+- `buttonColor=%23000000` to set the background color of the primary button to any css value
+- `inheritFont=true` to disable the widget font styling allowing it to inherit from the parent container
+- `buttonType=outline` to set the button type. Type `outline` (currently the only option) sets the `buttonColor` as a border around the button with a transparent background
+- `title=TITLE` to set a styled header to the widget
+- `textColor=red` to set the text color of the widget (template titles)
+- `button=BUTTONTEXT` to set the primary button text (default: Use This Zap)
+- `width=600` to set a fixed width to the widget. By default, the widget markup is fully responsive with a max width of 100% of its parent’s width.
+
+Include the additional options at the end of the script text with an ampersand, such as:
+
+```javascript
+<script src="https://zapier.com/apps/embed/widget.js?services=APP&borderColor=green&limit=10&theme=dark"></script>
+```
+
+You can load templates asynchronously (without blocking the rest of your page’s content) by adding the `async` attribute and `html_id` parameter to the script tag, like so:
+```javascript
+<div id="foo"></div>
+<script async src="https://zapier.com/apps/embed/widget.js?services=mailchimp&html_id=foo"></script>
+```
+
+> *Note:* The `html_id` parameter must point to a valid element `id` on the page you wish to be the direct parent of the embed.
+
+Using React or another client-side rendering library? See documentation on how to integrate with these here: [https://platform.zapier.com/partners/zap-templates#support-for-react](https://platform.zapier.com/partners/zap-templates#support-for-react).
+
+### Zap Manager Element
+![](https://zapier.com/partner/_next/image?url=%2Fpartner%2Fimg%2Fzap-manager%2Fpreview2-dark-large.png&w=1920&q=75)
```

Comment:
We should probably move this image to Zappy and link from there since it's much more likely to outlive this path. Same for any other image urls in this PR